src/QMCWaveFunctions/SPOSetBuilderFactory.cpp

```diff
@@ -301,37 +300,42 @@ void SPOSetBuilderFactory::build_sposet_collection(xmlNodePtr cur)
     collection_name = collection_type;
 
   app_summary() << std::endl;
-  app_summary() << "   Single particle orbitals (SPO) collections" << std::endl;
-  app_summary() << "   ------------------------------------------" << std::endl;
+  app_summary() << "   Single particle orbitals (SPO) collection" << std::endl;
+  app_summary() << "   -----------------------------------------" << std::endl;
   app_summary() << "    Name: " << collection_name << "   Type input: " << collection_type << std::endl;
   app_summary() << std::endl;
 
   // create the SPOSet builder
   SPOSetBuilder* bb = createSPOSetBuilder(cur);
+
+  // going through a list of basisset entries
```

Comment:
basis set is two words I think the cname should be "basis_set"

Not addressable here should we start to make a list of input names to update?

Reply:
I vote for not changing. It is short enough. Long ones need breakup. If change, we need proper protections.